examples/fastai_simple.py

```diff
@@ -0,0 +1,78 @@
+"""
+Optuna example that demonstrates a pruner for fastai.
+
+In this example, we optimize the hyperparameters of a convolutional neural network for hand-written
+digit recognition in terms of validation accuracy. The network is implemented by fastai and
+evaluated on MNIST dataset. Throughout the training of neural networks, a pruner observes
+intermediate results and stops unpromising trials.
+Note that this example will take longer than the other examples
+as this uses the entire MNIST dataset.
+
+You can run this example as follows, pruning can be turned on and off with the `--pruning`
+argument.
+    $ python fastai_integration.py [--pruning]
+"""
+
+import argparse
+from functools import partial
+
+from fastai.vision import ImageDataBunch
+from fastai.vision import Learner
+from fastai.vision import URLs
+from fastai.vision import accuracy
```

Comment:
Please reorder the imports as suggested by flake8.

https://circleci.com/gh/optuna/optuna/25969
```
#!/bin/bash -eo pipefail
. venv/bin/activate
flake8 .
./examples/fastai_simple.py:22:1: H306: imports not in alphabetical order (fastai.vision.urls, fastai.vision.accuracy)
1     H306: imports not in alphabetical order (fastai.vision.urls, fastai.vision.accuracy)
```